tests/unit/Messaging/MessagingTest.php

```diff
@@ -204,23 +207,28 @@ public function testFromPayloadCollectionLevelPermissions(): void
         $result = Realtime::fromPayload(
             event: 'databases.database_id.collections.collection_id.documents.document_id.create',
             payload: new Document([
-                '$id' => 'test',
-                '$collection' => 'collection',
-                '$read' => ['role:admin'],
-                '$write' => ['role:admin']
-            ]),
-            collection: new Document([
-                '$id' => 'collection',
-                '$read' => ['role:all'],
-                '$write' => ['role:all'],
-                'permission' => 'collection'
+                '$id' => ID::custom('test'),
+                '$collection' => ID::custom('collection'),
+                '$permissions' => [
+                    'read(admin)',
+                    'update(admin)',
+                    'delete(admin)',
```

Comment:
An `admin` permission ? I believe this was something we planned for future versions ?